smithy-openapi/src/main/java/software/amazon/smithy/openapi/fromsmithy/protocols/AbstractRestProtocol.java

```diff
@@ -398,10 +403,17 @@ private void createResponsePayload(
             String mediaType,
             Context<T> context,
             HttpBinding binding,
-            ResponseObject.Builder responseBuilder
+            ResponseObject.Builder responseBuilder,
+            Shape operationOrError
     ) {
+        Schema schema = context.inlineOrReferenceSchema(binding.getMember());
```

Comment:
Can you add a comment here (and in the other bit of updated code) to state why we're doing this?

Reply:
Added documentation in both places.